dgraphtest/image.go

```diff
@@ -86,13 +86,22 @@ func openDgraphRepo() (*git.Repository, error) {
 }
 
 func checkoutGitRepo(repo *git.Repository, hash *plumbing.Hash) error {
-	worktree, err := repo.Worktree()
-	if err != nil {
-		return errors.Wrap(err, "error while getting git repo work tree")
-	}
-	if err := worktree.Checkout(&git.CheckoutOptions{Hash: *hash, Force: true}); err != nil {
-		return errors.Wrap(err, fmt.Sprintf("error while checking out git repo with hash [%v]", hash.String()))
-	}
+	// For some reason, when we use the worktree.Checkout, it seems to fail a lot in CI with errors
+	//   test panicked: error while checking out git repo with hash [a77bbe8ae0d42697a38069a9749cfe71c2dafbe6]:
+	//   open /home/ubuntu/actions-runner/_work/dgraph/repo/ee/updatemanifest: no such file or directory
+	// Until we know why that is happening, I [Aman] have moved to use the git command line tool.
```

Comment:
Just make the comment as we, you don't have to take the responsibility. Maybe add a todo here saying we can upgrade this.

Reply:
It's no big deal. Thanks for the review.